_docs/ios/1.160.0.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: 1.160.0
+permalink: /docs/ios/1.160.0/
+parent: iOS
+nav_order: -160
+release_date: 22.11.2022
+---
+
+## Preview Feature - Bridge configuration page:
+### Updated designs
+Small design tweaks and user experience improvements.
+
+### New details about the current connection status
+Current network section was updated with information about signal strength (via icon), IP address, gateway address, mac address and in case of connection issue, information if bridge is disconnected from WiFi network or is unable to connect to tedee service.\
+![screenshot](/tedee-release-notes/docs/ios/assets/1.160.0-bridge-wifi-details.png)
+
+## Star icon added for "Favorite devices"
+We introduce "star" icon for devices marked as favorites, on my devices page, when favorites section is disabled. This icon also appears on device details page.\
+![screenshot](/tedee-release-notes/docs/ios/assets/1.160.0-device-star.png)
+
+## Star icon added for "Favorites" section
+Favorite section on my devices page got "star" icon.\
+![screenshot](/tedee-release-notes/docs/ios/assets/1.160.0-favorite-section-star.png)
+
+## You can now choose if a Favorite device is displayed before all other devices
+When favorites section is disabled, you can select if devices marked as favorites should appears on top of their section.\
+![screenshot](/tedee-release-notes/docs/ios/assets/1.160.0-favorites-settings.png)
+
+## Issue with the lack of information when user needed to log in again
+We fixed issue when user needs to sign in again but no information about it is present to the user.
+
+## Red dot is now present when a device indicates low battery while charging
```

Comment:
```
## Red dot is now presented when a device indicates low battery while charging
 When lock has less than 15% but it is charging, red dot wasn't presented in lock settings. Now, it will be presented until lock charge to more that 15%.
 ```